Userland/Libraries/LibJS/Tests/error-stack.js

```diff
@@ -0,0 +1,28 @@
+test("Anonymous function", function () {
+    let stack_string = (() => {
+        return Error();
+    })().stack;
+    console.log(stack_string);
+    let [header, ...stack_frames] = stack_string.split("\n");
+    // FIXME: Chrome does not generate a Stack frame for the call to Error()
```

Comment:
This FIXME probably belongs next to the implementation instead? Also, given that this is non-standard and apparently SM and JSC do different things, we don't need do mirror V8 exactly.